### PR TITLE
wallet class implemented

### PR DESCRIPTION
**Pull request by Riyuk-04**

### CODE REVIEW

File: src/adapter.ts [hunk 1]
```diff
@@ -1,16 +1,103 @@
...
+import {
+  ConnectFeature,
+  ConnectMethod,
+  EventsFeature,
+  EventsOnMethod,
+  SuiSignAndExecuteTransactionFeature,
+  SuiSignAndExecuteTransactionMethod
+} from "@mysten/wallet-standard";
 
 declare const window: {
```
Comment (MartianAbhishek) [inline src/adapter.ts]: please remove this, this is replacing actual martian object

File: src/adapter.ts [hunk 2]
```diff
@@ -1,16 +1,103 @@
...
   martian: { sui: MartianApis };
 };
 
-export class MartianWalletAdapter {
+export class MartianWalletAdapter implements Wallet{
+  icon!: WalletIcon;
+  chains: IdentifierArray = [SUI_DEVNET_CHAIN, SUI_TESTNET_CHAIN];
+  accounts!: readonly WalletAccount[];
```
Comment (MartianAbhishek) [inline src/adapter.ts]: make this empty array, it should fix register wallet not able to connect issues

Comment (Riyuk-04) [reply src/adapter.ts]: Done

File: src/adapter.ts (hunk 1 above)
Comment (Riyuk-04) [reply src/adapter.ts]: Deleting this gives error in lines 156, 160

Comment (MartianAbhishek) [reply src/adapter.ts]: Ah, i think we can declare type instead? Like type window {sui: any} something like this

**Review by MartianSiddharth — APPROVED** [review]

**Pull request merged**